_opensearch/query-dsl/index.md

```diff
@@ -12,16 +12,60 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to perform customized searches on your data. You can also combine query fields to perform more complex searches and find more granular data.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+OpenSearch supports the following query DSL query type categories:
+
+- **Compound queries** – Used to perform combined queries with the following query types:
+    - **Boolean** `bool` – Combines several simple query terms by Boolean clause rules that act as logical operators. To learn more, see [Boolean queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/bool/).
+    - **Constant score** `constant_score` – Provides a `filter` query with the relevance score as a floating point number assigned by the `boost` field. Acts as a wrapper for the filter.
+    - **Disjunction max** `dis_max` – Returns documents that match one or more queries that are wrapped together by clauses. If a document matches multiple query clauses, it gets the highest relevant score assigned to it.
+    - **Function score** `function_score` – Modifies the score of documents that are returned by a query. You define a query and one or more functions to compute a score for each document that matches the query.
+
+- **Full-text queries** – Used to search documents for one or more terms and filter with advanced options. These queries perform text analysis and return detailed information, including relevance scores for each document match. To learn more, see [Full-text queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/).
+
+- **Geographic and cartesian shape queries** – Used to search documents that include geographic geopoint field data. Also used to search two-dimensional geometries that map out cartesian data, such as the `xy_point` field that supports x and y pairs and the `xy_shape` field that supports points, lines, circles, and polygon shapes. To learn more, see [Geographic and shape queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/geo-and-shape/).
+
+- **Joining queries** – Used to perform `nested`, `has_child`, `has_parent`, and `parent_id` queries.
+
+- **Query string queries** – Used to perform complex queries for a string with multiple optional fields using `query_string` syntax. To learn more, see [Query string queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/query-string/).
```

Comment:
The first sentence here is a bit confusingly written. Can we simplify/clarify?

Reply:
Query string query should go under "full text queries". It's a subtype of full text.
```suggestion
- **Query string queries** – Query string queries use a more concise `query_string` syntax. They can incorporate wildcards and search multiple fields. To learn more, see [Query string queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/query-string/).
```